### ISSUE TITLE
Unrecognized text on build

### ISSUE THREAD

**Issue by josh-teichro** [LucidNinja/tailwind-container-break-out#6]
This is related to #3, which I was also getting. i.e.

```
Lexical error on line 1: Unrecognized text.
```

Basically, if you define your container padding for some but not for all screen widths in your `tailwind.config.js`, the screen width missing a padding value will output 'undefined' in the calc, e.g.

```
(-100vw + var(--twcb-scrollbar-width)) / 2 + 768px / 2 - undefined
``` 

There were no warnings or anything though, it just failed to build. Strangely though, only 'production' builds failed for me (I guess some configs dont care about the 'undefined' value?). 

A temporary solution though is to just define your container padding for all breakpoints, but Tailwind allows you to skip some if they're the same as the previous breakpoint, so I believe this should behave the same.

**Comment by dalepgrant** [comment]
Also experiencing this, thanks for the temporary solution @josh-teichro.

**Comment by RichardBeno** [comment]
Thanks @josh-teichro I ran into the same issue. Declaring the container padding for all breakpoints like you described resolved it for now 👍

**Comment by LucidNinja** [comment]
This has been fixed in version [2.0.7](https://github.com/LucidNinja/tailwind-container-break-out/releases/tag/v2.0.7)
